onnx/helper.py

```diff
@@ -922,7 +922,7 @@ def make_attribute(  # pylint: disable=too-many-statements
 
     if attr_type is not None and attr.type != attr_type:
         raise TypeError(
-            f"Inferred attribute type {attr.type} mismatched with specified type {attr_type}"
+            f"Inferred attribute type {attr_type_to_str(attr.type)}({attr.type}) mismatched with specified type {attr_type_to_str(attr_type)}({attr_type})"
```

Comment:
```suggestion
            f"Inferred attribute type '{attr_type_to_str(attr.type)}' ({attr.type}) mismatched with specified type '{attr_type_to_str(attr_type)}' ({attr_type})"
```

Quote the user input for clarity.